ee/modules/610-service-with-healthchecks/crds/doc-ru-servicewithhealthchecks.yaml

```
@@ -271,55 +266,57 @@ spec:
                   description: |-
-                    `publishNotReadyAddresses` указывает, что любой агент, который обрабатывает конечные точки для этого сервиса, должен игнорировать любые указания на готовность/неготовность. 
-                    Основной случай использования установки этого поля — это сервис `StatefulSet` для распространения SRV DNS-записей для его подов с целью обнаружения пиров. 
+                    `publishNotReadyAddresses` указывает, что любые агенты, занимающиеся обработкой конечных точек данного сервиса, должны игнорировать статусы готовности или неготовности. 
+                    Основное применение этого поля заключается в том, чтобы сервисы `StatefulSet` могли распространять записи SRV DNS для своих подов с целью обнаружения пиров. 
                     
-                    Контроллеры Kubernetes, которые генерируют ресурсы `Endpoints` и `EndpointSlice` для сервисов, интерпретируют это как указание на то, что все конечные точки считаются "готовыми", даже если сами поды не готовы. 
-                    Агенты, которые потребляют только конечные точки, созданные Kubernetes, через ресурсы `Endpoints` или `EndpointSlice`, могут с уверенностью предполагать это поведение.
+                    Контроллеры Kubernetes, которые генерируют ресурсы `Endpoints` и `EndpointSlice` для сервисов, трактуют это как указание, что все конечные точки следует считать "готовыми", даже если поды не прошли пробы и не готовы.
                 selector:
                   description: |-
-                    Маршрутизировать трафик сервиса к подам с ключами и значениями меток, соответствующими этому селектору.
-                    Если селектор пуст или отсутствует, предполагается, что у сервиса есть внешний процесс, управляющий его конечными точками, которые Kubernetes не будет изменять.
+                    `Selector` маршрутизирует трафик сервиса к подам, у которых ключи и значения меток соответствуют этому `selector`'у.
+                    Если `selector` пуст или отсутствует, предполагается, что у сервиса есть внешний процесс, управляющий его конечными точками, которые Kubernetes не будет изменять.
                     
-                    Это применяется только к типам ClusterIP, NodePort и LoadBalancer.
+                    Применимо только к типам ClusterIP, NodePort и LoadBalancer.
                     
                     Игнорируется, если тип — ExternalName.
                     [Подробнее.](https://kubernetes.io/docs/concepts/services-networking/service/)
                 sessionAffinity:
                   description: |-
                     Поддерживает `ClientIP` и `None`. Используется для поддержания сессий с привязкой к клиенту.
-                    Включает сессию с привязкой к IP-адресу клиента.
-                    Должно быть либо `ClientIP`, либо `None`.
-                    
-                    По умолчанию — `None`.
+                    Должно быть указано значение либо `ClientIP`, либо `None`.
```

```suggestion
                    Должно быть указано значение либо `ClientIP`, либо `None`.
```
Тут надо посмотреть сгенерированную спеку - там возможно есть уже дефолт.